mfr/extensions/pdf/export.py

```diff
@@ -0,0 +1,85 @@
+import os
+
+import imghdr
+from PIL import Image
+
+from reportlab.pdfgen import canvas
+
+from mfr.core import extension
+from mfr.extensions.pdf import exceptions
+
+
+class PdfExporter(extension.BaseExporter):
+
+    def __init__(self, *args, **kwargs):
+
+        super().__init__(*args, **kwargs)
+        self.metrics.add('pil_version', Image.VERSION)
+
+    def tiff_to_pdf(self, tiff_img, max_size):
+        max_pages = 40
+        width, height = tiff_img.size
+        c = canvas.Canvas(self.output_file_path)
+        c.setPageSize((max_size[0], max_size[1]))
+
+        page = 0
+
+        # This seems to be the only way to write this loop at the moment
+        while True:
+            try:
+                tiff_img.seek(page)
+            except EOFError:
+                break
+
+            # Center the image and draw it in the canvas
```

Comment:
Be consistent on comments style (a few occurrences).
* should we capitalize the first letter of the first word (whichever you prefer)
* should we add punctuation at the end of the line (usually not)